article/detorakis-2016.tex

```diff
@@ -435,159 +470,80 @@ \section{Methods}\label{methods}
-given in Table~\ref{Table:5} and the corresponding input (see
-Table~\ref{Table:3}). 
-The first test case is the simulation of burst responses. In this case, we 
-examined if the model is able to generate different types of burst responses
-such as in Fig\footnote{From now and then all the figures of the original
-article will be referred as Fig}~$1$ of the original article. Thus, we applied
-a periodic current pulse of $-1 \mu A/cm^2$ amplitude at several different 
-frequencies ($\frac{1}{P_0}$ is the frequency in $Hz$ and $P_0$ is the
-corresponding period in $ms$) ranging from $0.1Hz$ to $15Hz$ with a resolution
-(discretization) of $150$ points. The same number of samples used for 
-discretizing the duration for each frequency. The results are shown in 
-Figure~\ref{Fig:1} left panel.
-Our right panel is slightly different from the original one. In order to produce
-this diagram, we simulated the model using different frequencies and duration
-pulses and we counted spikes in one period of each iteration (one frequency out
-of $150$ and one duration out of $150$). The author in \cite{wang:1994} shows
-in Fig~$1$ a shaded are where the number of spikes is $0.5$. By simple counting
-this is impossible. However, if one takes the average of suprathreshold and 
-subthreshold spikes in one period can obtain some numbers like these.  
-
-In addition, we picked up three specific frequencies 
-$\frac{1}{P_0}=10, 5, 0.5$ and pulse duration $p=10, 120, 1200ms$
-($\frac{p}{P_0} = 0.1, 0.6, 0.6$) in order to test if the model can reproduce
-the upper panel of Fig~$1$ of \cite{wang:1994}. The first frequency that we
-used is not used in the original article, so we do not further discuss it. 
-For the two other frequencies, we have that the corresponding periods in $ms$
-are $200ms$ and $2000ms$, respectively. Now, if we compute the ON duration
-of each pulse (this is the $p$), we get $p = P_0 \times 0.6$ and this gives
-$p = 120ms$ ($5Hz$) and $p=1200ms$ ($0.5Hz$). These numbers are different
-from the ones given in the caption of Fig~$1$ in \cite{wang:1994}. 
-Also according to the parameters diagram in the left panel original
-of Fig~$1$, the frequency $frac{1}{P_0} = 5Hz$ and the ratio $\frac{p}{P_0} =
-0.6$ generate a burst of two ($2$) spikes. This is in contradiction, with
-what is presented in the same figure on the upper right panel, where 
-there is a burst of one ($1$) spike. This does not happen in our 
-implementation, where the results on the right panels of Fig~\ref{Fig:1}
-are in agreement with the parameters diagram on the left panel. 
+given in Table~\ref{Table:5} and the corresponding input (see 
+Table~\ref{Table:3}). First, we examined what is the response of the reference 
+implementation to rhythmic hyperpolarization. In \cite{wang:1994} this is 
+illustrated in Fig~$1$\footnote{From now and then all the 
+    figures of the original article will be referred as Fig} of the original
+article. 
```

Comment:
This is double-up: "In Wang (1994) ... of the original article". Wang is the original article.